src/Microsoft.Health.Fhir.Core/Features/Conformance/Models/ListedResourceComponent.cs

```diff
@@ -5,21 +5,23 @@
 
 using System;
 using System.Collections.Generic;
+using Microsoft.Health.Fhir.Core.Configs;
 using Microsoft.Health.Fhir.Core.Models;
 using Microsoft.Health.Fhir.ValueSets;
 
 namespace Microsoft.Health.Fhir.Core.Features.Conformance.Models
 {
     public class ListedResourceComponent
     {
-        public ListedResourceComponent()
+        public ListedResourceComponent(CoreFeatureConfiguration configuration)
```

Comment:
@brendankowitz I ended up keeping the config info in this class and passing it into the `DefaultHashSet` constructor. This was a little cleaner than adding a setter to the `DefaultHashSet` class.